t/file-filecopied2.neon

```diff
@@ -0,0 +1,15 @@
+IMPORT file
+
```

Comment:
Create a dummy file here (as in filecopied1) so that you can test that the overwrite actually does overwrite an existing file here.